src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -689,8 +711,12 @@ public Tensor cpu()
             /// Returns a copy of this object in CUDA memory.
             /// If this object is already in CUDA memory and on the correct device, then no copy is performed and the original object is returned.
             /// </summary>
-            public Tensor cuda()
+            public Tensor cuda(Device? device = null)
             {
+                if (device is null) {
+                    device = torch.CUDA;
```

Comment:
This value does not seem to be used anywhere.

Reply:
Thanks for catching that!